### ISSUE TITLE
mask preprocess

### ISSUE THREAD

**Issue by Nomination-NRB** [luxiaolili/IDM-VTON-train#5]
May I ask why the mask is not used after preprocessing, but obtained from sample and interpolated again? Is this mask preprocessing still necessary?

```python
# line 446
mask = mask_processor.preprocess(sample["mask"], height=args.height, width=args.width, crops_coords=None, resize_mode="default")
mask = F.interpolate(sample['mask'].to(torch.float32), size=(int(args.height/8), int(args.width/8)))
mask = mask.to(accelerator.device, dtype=torch.float16)
```

**Comment by luxiaolili** [comment]
> May I ask why the mask is not used after preprocessing, but obtained from sample and interpolated again? Is this mask preprocessing still necessary?
> 
> ```python
> # line 446
> mask = mask_processor.preprocess(sample["mask"], height=args.height, width=args.width, crops_coords=None, resize_mode="default")
> mask = F.interpolate(sample['mask'].to(torch.float32), size=(int(args.height/8), int(args.width/8)))
> mask = mask.to(accelerator.device, dtype=torch.float16)
> ```

no need for mask_processor,  I delete it
